packages/radio-group/src/LionRadioGroup.js

```diff
@@ -24,102 +25,26 @@ import { LionFieldset } from '@lion/fieldset';
  * @extends {LionFieldset}
  */
 
-export class LionRadioGroup extends LionFieldset {
-  get checkedValue() {
-    const el = this._getCheckedRadioElement();
-    return el ? el.modelValue.value : '';
-  }
-
-  set checkedValue(value) {
-    this._setCheckedRadioElement(value, (el, val) => el.modelValue.value === val);
-  }
-
-  get serializedValue() {
-    const el = this._getCheckedRadioElement();
-    return el ? el.serializedValue : '';
-  }
-
-  set serializedValue(value) {
-    this._setCheckedRadioElement(value, (el, val) => el.serializedValue === val);
-  }
-
+export class LionRadioGroup extends ChoiceGroupMixin(LionFieldset) {
   get formattedValue() {
-    const el = this._getCheckedRadioElement();
+    const el = this._getCheckedElements();
     return el ? el.formattedValue : '';
   }
 
   set formattedValue(value) {
```

Comment:
Why does `formattedValue` exist only on `LionRadioGroup`? Should we implement it for `LionCheckboxGroup` as well? Does it even make sense for this property to exist?

Reply:
The setter will set checked to the element of which the formattedValue is the same as the group's formattedValue, but to me this makes no sense for choice inputs/groups, because you don't actually format the user input, you simply respond to the user selection with a visual element (the radio / checkbox). So I'm personally fine with removing it, since it's also untested feature..